config/discourse_defaults.conf

```diff
@@ -390,8 +390,11 @@ log_line_max_chars = 160000
 # this value is included when generating static asset URLs.
 # Updating the value will allow site operators to invalidate all asset urls
 # to recover from configuration issues which may have been cached by CDNs/browsers.
-asset_url_salt = 
+asset_url_salt =
 
 # disable service-worker caching. An 'offline' page will still be used as an offline fallback.
 # This flag is for a temporary experiment, and may be removed at any time
 disable_service_worker_cache = true
+
+# Sets cache sizes. This many sites can have the maximum number of keys in each LiveCache
```

Comment:
"Sets cache sizes" is slightly confusing to me because the setting controls the number of sites that can have the maximum number of keys. From what I can tell, the number of keys is not set by this config option.

Reply:
Each cache has a global size and a per-site size. This is the multiple between them.

Reply:
Ic in this case. I don't think we should expose this configuration option. Instead, I think we should just set this to the number of multisites configured by RailsMultisite.

Reply:
If you set it to the number of sites, the global limits will never be invoked. It needs to be less that that to be useful.

Reply:
If that is the case, how did we arrive at 200 as the default? For clusters which has less than 200 sites, doesn't it also mean that the global limit will never be hit? Maybe we need to it to be a fraction of the total number of sites instead?